addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Hi @daveoxley,

Thanks for getting back to us. I'm not entirely sure why @talltechdude decided to copy the source into the project rather than pull from maven. Unfortunately, Scott hasn't been around for sometime. Recently, @soverton100 was going to make some alterations/simplifications. 

@kaikreuzer is in a better position to offer advice on licensing.